### PR TITLE
Update manifest and changes file for release

### PR DESCRIPTION
**Pull request by jscarty**

### CODE REVIEW

**Comment by pvoglhofer** [comment]
should probably wait until the new Proxy class is merged.

File: Makefile.PL
Comment (pvoglhofer) [inline Makefile.PL:33]: Gtk3::WebKit2 is on 0.012

File: Changes
Comment (pvoglhofer) [inline Changes:3]: This should be 0.12 now.

**Review by pvoglhofer — CHANGES_REQUESTED** [review]

File: Makefile.PL
Comment (jscarty) [reply Makefile.PL:33]: Gtk3::Webkit2 0.012 only includes text changes. 0.011 had some needed changes for the release (seemed to cause problems building on cpan). It doesn't need the newer version.

**Review by pvoglhofer — APPROVED** [review]

**Pull request merged**
